src/app/public/features/components/repository/profile/profile.component.html

```diff
@@ -1 +1,51 @@
-<p>profile works!</p>
+<div class="card" style="">
+  <div class="container">
```

Comment:
and following my previous comment... here i would add  a class like: `class="container ProfileCard__container"` (following the BEM convention)